cpp/src/arrow/engine/substrait/protobuf_test_util.cc

```diff
@@ -0,0 +1,32 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#include "arrow/engine/substrait/protobuf_test_util.h"
+
+#include <google/protobuf/stubs/common.h>
```

Comment:
in the docs, it seems ShutdownProtobufLibrary comes from message_lite.h; stubs/common.h isn't in the API documentation and I'm not sure we should depend on an internal-looking header

Reply:
good catch David, I'll change that part, thanks.